src/pybind/mgr/orchestrator/module.py

```diff
@@ -387,6 +396,13 @@ def _get_hosts(self, format: Format = Format.plain) -> HandleCommandResult:
                 table.add_row((host.hostname, host.addr, ' '.join(
                     host.labels), host.status.capitalize()))
             output = table.get_string()
+        output += f'\n{len(hosts)} hosts in cluster'
+        if label:
+            output += f' who had label {label}'
+        if matches_name:
+            output += f' whose hostname matched {matches_name}'
+        if host_status:
+            output += f' with status {host_status}'
```

Comment:
pleas make sure you avoid adding this for json/yaml output

Reply:
Also found by @batrick . See https://tracker.ceph.com/issues/53452

Reply:
Oh good! I think you know where I found that :)

Reply:
Changed this to only appear on plain format.

Not sure if necessary, added "Fixes: https://tracker.ceph.com/issues/53452" to the commit